docs/solutions/integration-issues/background-widget-refresh-engine-plugin-and-safe-fallback-20260224.md

```
@@ -0,0 +1,64 @@
+symptoms:
+  - "Schedule-change notifications were shown while Android widgets stayed stale until app open"
+  - "Background schedule update path depended on Activity-bound widget channel wiring"
+  - "Background channel failures could surface as non-PlatformException and break task flow"
+root_cause: activity_bound_widget_channel_with_background_exception_gap
+severity: high
+tags: [android-widget, background-task, workmanager, method-channel]
+---
+
+# Fix: Android Widget Refresh After Background Schedule Updates
+
+## Problem
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)